cmd/ocm/gcp/gcp-client-shim.go

```diff
@@ -90,42 +91,55 @@ func (c *shim) CreateWorkloadIdentityPool(
 		}
 	}
 
+	// Enable the pool if it exists but is disabled.
+	if resp.Disabled {
+		if err := c.gcpClient.EnableWorkloadIdentityPool(
+			ctx,
+			resp.Name,
+		); err != nil {
+			return err
```

Comment:
should this not be wrapped similar to line #90

Reply:
ack.